extra-scientific/bedtools/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=bedtools
+PKGSEC=science
+PKGDES="A powerful tool for genome arithmetic"
```

Comment:
Seems like that only the make test requires samtools as described in Gentoo's ebuild. It just makes on my instance so I don't know if there would be any other dependencies...